src/bin/migrate.rs

```diff
@@ -448,6 +448,21 @@ fn migrations() -> Vec<Migration> {
             try!(tx.execute("DROP INDEX index_keywords_lower_keyword", &[]));
             Ok(())
         }),
+        Migration::add_column(20150715170350, "crate_owners", "owner_kind",
+                              "INTEGER NOT NULL DEFAULT 0"),
+        Migration::run(20150804170127,
+            "ALTER TABLE crate_owners ALTER owner_kind DROP DEFAULT",
+            "ALTER TABLE crate_owners ALTER owner_kind SET DEFAULT 0",
+        ),
+        Migration::add_table(20150804170128, "teams", "
+            id            SERIAL PRIMARY KEY,
+            name          VARCHAR NOT NULL UNIQUE,
```

Comment:
Can you make sure there's an index on this field? You're doing a query on it sometimes so it'll need to be indexed, although I forget if the `UNIQUE` constraint adds an index or not...